### PR TITLE
Fix #98

### LINKED ISSUE #98: Autocomplete add new option 'create' -> does not work for me
Hi there, I'm facing an issue, that the create option for the autocomplete field is not shown.
Saw the issue #91 but unfortunately I'm not dealing with self written templates. I'm using the default ones. Have no idea how I get that working.

image

image

"alterphp/easyadmin-extension-bundle": "^2.1",
"easycorp/easyadmin-bundle": "^2.0",

Any help is appreciated.

Regards
Markus

### PR DESCRIPTION
**Pull request by gonzakpo**
tested in chrome and firefox

Autocomplete add new option 'create' -> does not work for me #98

### CODE REVIEW

**Comment by alterphp** [comment]
@markus-fischbacher Can you confirm this solves your issue ?

**Comment by markus-fischbacher** [comment]
I‘ll test it tomorrow. 
Keep you posted.
Thanks!

**Comment by markus-fischbacher** [comment]
It is working using Chrome and Firefox. 

image

However, the rendering of the modal is not looking the way it should.

image

EDGE and IE are failing. But that's what @gonzakpo already told.

Thanks for the fix!

**Comment by gonzakpo** [comment]
> It is working using Chrome and Firefox.
> 
> image
> 
> However, the rendering of the modal is not looking the way it should.
> 
> image
> 
> EDGE and IE are failing. But that's what @gonzakpo already told.
> 
> Thanks for the fix!

There is an error in loading JS files, there will be see how to improve.

**Comment by alterphp** [comment]
@gonzakpo Can you try to fix the "bad rendering" and IE/Edge bug in the PR ?

**Comment by markus-fischbacher** [comment]
EasyAdminBundle in general is not working in IE, at least for my project.
In EDGE it is working, but the autocomplete is not rendered correctly.
It's looking that way:
image

**Comment by gonzakpo** [comment]
Yes.
The error starts before:

SCRIPT1028: SCRIPT1028: Expected identifier, string or number
app.js (26,27191)

 SCRIPT5009: SCRIPT5009: '$' is not defined
admin (5702,9)

 SCRIPT5009: SCRIPT5009: 'jQuery' is not defined
es.js (3,13)

 SCRIPT5009: SCRIPT5009: '$' is not defined
admin (5749,3)

 SCRIPT1028: SCRIPT1028: Expected identifier, string or number
bootstrap-all.js (64,85736)

 SCRIPT5009: SCRIPT5009: 'jQuery' is not defined
bootstrap-datepicker.min.js (7,73)

2 SCRIPT5009: SCRIPT5009: '$' is not defined
easyadmin-extension.js (55,3)

File: README.md [hunk 1]
```diff
@@ -81,7 +81,6 @@ easy_admin:
     design:
         assets:
             css:
-                - 'bundles/easyadmin/app.css'
```
[the remark this answers is not in the document]
Comment (gonzakpo) [reply README.md]: it is not necessary, it loads it by default

[the remark this answers is not in the document]
Comment (alterphp) [reply README.md]: unrelated to the PR, please don't push unrelated modifications.

**Comment by alterphp** [comment]
Thanks @gonzakpo for taking care of this !

**Pull request merged**